junction/proposals/templatetags/proposal_filters.py

```diff
@@ -11,3 +11,12 @@
 @register.filter(name='reviewer_comments')
 def reviewer_comments(proposal, user):
     return proposal.get_reviewer_comments_count(user) > 0
+
+
+@register.filter(name='has_content_urls')
+def get_content_urls(proposal):
+    if proposal.content_urls:
+        url = proposal.content_urls.split()
```

Comment:
The content urls may have something like `TBD`. In this case this will pass. Ideally it should only return all urls. Here is thread to extract all links in a paragraph https://stackoverflow.com/questions/4696418/regex-to-extract-all-urls-from-a-page.